grpc-bridge/src/main/java/dev/resteasy/grpc/bridge/generator/ServiceGrpcExtender.java

```diff
@@ -142,12 +144,16 @@ private void classHeader(Scanner scanner, StringBuilder sb, String fileName) {
     }
 
     private void imports(Scanner scanner, StringBuilder sb, String fileName) {
-        sb.append("import com.google.protobuf.Descriptors.FieldDescriptor;" + LS)
+        sb.append("import static dev.resteasy.grpc.bridge.runtime.Constants.ANY;" + LS)
+                .append("import com.google.protobuf.Descriptors.FieldDescriptor;" + LS)
                 .append("import com.google.protobuf.GeneratedMessageV3;" + LS)
                 .append("import com.google.protobuf.Timestamp;" + LS)
+                .append("import dev.resteasy.grpc.bridge.runtime.Utility;" + LS)
                 .append("import io.grpc.stub.StreamObserver;" + LS)
                 .append("import java.io.ByteArrayInputStream;" + LS)
                 .append("import java.io.ByteArrayOutputStream;" + LS)
+                .append("import java.io.InputStream;" + LS)
+                .append("import java.lang.reflect.Method;" + LS)
```

Comment:
Are these needed? Again, I don't see any usage being added.

Reply:
It looks like you're right. I see a few others that aren't used. Maybe they were used once upon a time.

Reply:
Removed